x/posts/keeper/common_test.go

```diff
@@ -131,6 +129,11 @@ func (suite *KeeperTestSuite) SetupTest() {
 		scopedIBCKeeper,
 	)
 
+	suite.sk = subspaceskeeper.NewKeeper(
+		suite.storeKey,
```

Comment:
It should be `keys[subspacetypes.StoreKey]`, or `subspacetypes.StoreKey` should not be created in `NewKVStoreKeys`.